README.Rmd

```diff
@@ -57,130 +55,169 @@ library(sfheaders)
 library(blockCV)
 library(yardstick)
 library(snakecase)
+library(patchwork)
```

Comment:
Not new with this PR, but I found this list of packages sort of overwhelming when I first pulled up the README.

How many of these packages are just loaded for one function and would likely consistently be used in these types of analysis? We could "re-export" relevant functions from those packages so that calling `library(forestecology)` would load most of what's needed. :-)